src/views/views.js

```diff
@@ -56,16 +72,14 @@ export const TREE = 'Tree'
  * @type {Map<string, CylcView>}
  */
 export const allViews = new Map([
-  [TREE, { component: TreeView, icon: mdiFileTree }],
-  ['Table', { component: TableView, icon: mdiTable }],
-  ['Graph', { component: GraphView, icon: mdiGraph }],
-  ['Log', { component: LogView, icon: mdiFileDocumentMultipleOutline }],
-  ['Analysis', { component: AnalysisView, icon: mdiChartLine }],
-  ['Gantt', { component: GanttView, icon: mdiChartGantt }]
+  ...workspaceViews,
+  ['Info', { component: InfoView, icon: mdiInformationOutline }],
 ])
```

Comment:
Note uses of `allViews` elsewhere. The info view shouldn't be selectable as the default view in settings: 
![image](https://github.com/user-attachments/assets/528bcb42-6cb0-4be7-b6ac-da8badedc963)